service-request-infno-spec-1.0.0-swagger.json

```diff
@@ -0,0 +1,213 @@
+    "title" : "Service Request Info API",
+    "license" : {
+      "name" : "Apache 2.0",
+      "url" : "http://www.apache.org/licenses/LICENSE-2.0.html"
+    }
+  },
+  "tags" : [ {
+    "name" : "3rd_parties"
+  }, {
+    "name" : "media"
+  } ],
+  "paths" : {
+    "/service-request/status/" : {
+      "get" : {
+        "tags" : [ "3rd_parties" ],
+        "summary" : "Get service request API Info",
+        "parameters" : [ {
+          "in" : "query",
+          "name" : "service_request_no",
+          "description" : "maximum number of records to return",
+          "required" : true,
+          "schema" : {
+            "type" : "integer",
+            "format" : "int32",
+            "minimum" : 0,
+            "maximum" : 9
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "OK",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "type" : "array",
+                  "items" : {
+                    "$ref" : "#/components/schemas/ServiceRequestSchema"
+                  }
+                }
+              }
+            }
+          },
+          "400" : {
+            "description" : "bad input parameter"
+          },
+          "404" : {
+            "description" : "Page Not Found"
+          },
+          "500" : {
+            "description" : "Internal Server Error"
+          },
+          "502" : {
+            "description" : "Bad Gateway Error"
+          },
+          "503" : {
+            "description" : "Service Unavailable Error"
+          }
+        }
+      }
+    },
+    "/service-request/stats/" : {
+      "get" : {
+        "tags" : [ "3rd_parties" ],
+        "parameters" : [ {
+          "in" : "query",
+          "name" : "area",
+          "required" : false,
+          "schema" : {
+            "type" : "string"
+          }
+        }, {
+          "in" : "query",
+          "name" : "type_identifier",
+          "required" : false,
+          "schema" : {
+            "type" : "string"
+          }
+        } ],
+        "responses" : {
+          "200" : {
+            "description" : "OK",
+            "content" : {
+              "application/json" : {
+                "schema" : {
+                  "type" : "array",
+                  "items" : {
+                    "$ref" : "#/components/schemas/AreaSchema"
+                  }
+                }
+              }
+            }
+          },
+          "400" : {
+            "description" : "bad input parameter"
+          },
+          "404" : {
+            "description" : "Page Not Found"
+          },
+          "500" : {
+            "description" : "Internal Server Error"
+          },
+          "502" : {
+            "description" : "Bad Gateway Error"
+          },
+          "503" : {
+            "description" : "Service Unavailable Error"
+          }
+        }
+      }
+    },
+    "/service-request/image_placeholder/" : {
```

Comment:
rather make this path, `image_classification`

Reply:
path changed